bioblend/galaxy/datasets/__init__.py

```diff
@@ -157,20 +177,104 @@ def get_datasets(self, limit=500, offset=0, history_id=None):
           For example, if ``limit`` is set to 100 and ``offset`` to 200,
           datasets 200-299 will be returned.
 
+        :type name: str
+        :param name: Dataset name to filter on.
+
+        :type extension: str or list of str
+        :param extension: Dataset extension (or list of extensions) to filter on.
+
+        :type state: str or list of str
+        :param state: Dataset state (or list of states) to filter on.
+
+        :type visible: bool
+        :param visible: If ``False``, include datasets marked as not ``visible``.
+
         :type history_id: str
         :param history_id: Encoded history ID to filter on.
 
+        :type create_time_min: str
+        :param create_time_min: Show only datasets created after the provided
+          time and date, which should be formatted as ``YYYY-MM-DDTHH-MM-SS``.
+
+        :type create_time_max: str
+        :param create_time_max: Show only datasets created before the provided
+          time and date, which should be formatted as ``YYYY-MM-DDTHH-MM-SS``.
+
+        :type update_time_min: str
+        :param update_time_min: Show only datasets last updated after the provided
+          time and date, which should be formatted as ``YYYY-MM-DDTHH-MM-SS``.
+
+        :type update_time_max: str
+        :param update_time_max: Show only datasets last updated before the provided
+          time and date, which should be formatted as ``YYYY-MM-DDTHH-MM-SS``.
+
+        :type order_by: str
+        :param order_by: Attribute for ordering datasets - must be one of
+          ``create_time`` (default), ``update_time``, ``hid``, ``name``.
```

Comment:
The Galaxy docs for this were wrong..., see https://github.com/galaxyproject/galaxy/pull/11933 , so `extension` and `history_id` are missing.
Additionally, "Orders can be stacked as a comma-separated list of values", so I am leaning towards just pass the value through to Galaxy for validation (dropping `order_descending` as well). What do you think?

Reply:
Thanks for spotting that!

> Orders can be stacked as a comma-separated list of values

I did notice that, I just didn't think it was necessary to support multiple ordering attributes in BioBlend. But if you disagree, I don't have any objection to just letting the user pass whatever they like to `order`.

Reply:
> > Orders can be stacked as a comma-separated list of values
> 
> I did notice that, I just didn't think it was necessary to support multiple ordering attributes in BioBlend. But if you disagree, I don't have any objection to just letting the user pass whatever they like to `order`.

I bet there's going to be a user asking for the multiple ordering :laughing: . I'd be in favour of letting the user pass their preferred value.